docs/plugins/spectate.rst

```diff
@@ -11,7 +11,30 @@ Usage
 
 ::
 
-    enable spectate
+    spectate
+    spectate <option>
```

Comment:
Requiring an explicit command to stop spectate mode seems heavy handed. How about this:
`spectate [--nopause]` starts following dwarves, if `--nopause` is specified, game events (e.g. forgotten beast attacks or cavern/magma discoveries) will not cause the game to pause. The player can still pause manually and interact with the game.

Stop spectate mode by moving the game cursor while the game is unpaused.

Reply:
Yeah, try it out and see if it is disruptive. I do strongly feel that the default behavior should match what the DF follow functionality does. Without market research it's hard to say for certain, but spectate is similar enough to `f`ollow that players will likely expect the "cancel" behavior to also be similar.

Reply:
So, it occurred to me I've already started a methodology for how to implement this when I added `auto-unpause`. I'll just add an `auto-disengage` as well. Logically I think they probably start turned off by default but I can do the other way if it matters to you. I may keep the configuration persistent like I think I saw skimming through autonestbox, on the fence cause then I'd have to actually require setting a state.. but that's probably a good thing to require

Reply:
alternatively if you can tell me what I'd name the oppositely defaulted setting which does the same thing as auto-disengage I can always name it that and set the default to disengage by default. I cannot think of what that would be called.